docgen/content-sources/toc.yaml

```diff
@@ -0,0 +1,75 @@
+toc:
+- title: "functions"
+  path: /docs/reference/functions/functions
+
+- title: "functions.analytics"
+  path: /docs/reference/functions/functions.analytics
+  section:
+  - title: "AnalyticsEvent"
+    path: /docs/reference/functions/AnalyticsEvent
```

Comment:
@hiranya911 this seems to work when generating from `/src` in the functions SDK, but it's not the nice hierarchy that I saw in admin. I'd love for these paths to be namespaced better but I suppose it doesn't matter since these paths just make the links work, right?